nicolegooden/indecision-maker
# Fix code based on linter warnings

#### What's this PR do?  

+ Fixes code in test files based on linter warnings (mostly `it block` descriptions)

#### Where should the reviewer start? 

Start in any relevant test files.

#### How should this be manually tested?  

Run `npm test` to ensure the tests that are expected to pass are indeed passing.

#### Any background context you want to provide?  

Just wanted to clean this up!

#### Screenshots 

![Screen Shot 2020-11-04 at 9 56 51 PM](https://user-images.githubusercontent.com/62262404/98200010-de41ce80-1ee9-11eb-9286-e19fc6042f03.png)
